src/ui/TimeTable.tsx

```diff
@@ -296,13 +296,21 @@ const TimeTable = ({ id, className, onChange, value, ...props }: Props) => {
     };
   };
 
+  const nextWeekdayDate = (dayInWeek: number): Date => {
```

Comment:
date-fns has a buildin function for this 
https://date-fns.org/v2.27.0/docs/nextWednesday
or 
https://date-fns.org/v2.27.0/docs/nextDay

Reply:
Thx! Nice when life can be so easy :)